test/unit/spec/PasswordExpired_spec.js

```diff
@@ -66,18 +66,18 @@ function (Okta, OktaAuth, LoginUtil, Util, PasswordExpiredForm, Beacon, Expect,
     return Expect.waitForPasswordExpired(settings);
   }
 
-  function setupWarn (numDays) {
+  function setupWarn (settings, numDays) {
```

Comment:
Should we have settings as the param after `numDays` so we don't have to pass in `undefined` in tests below?